index.js

```diff
@@ -141,10 +162,9 @@ module.exports = results => {
       if (x.type === 'message') {
         const line = [
           '',
-          x.severity === 'warning' ? logSymbols.warning : logSymbols.error,
+          logSymbols.warning,
           ' '.repeat(maxLineWidth - x.lineWidth) + chalk.dim(x.line + chalk.gray(':') + x.column),
-          ' '.repeat(maxColumnWidth - x.columnWidth) + x.message,
-          ' '.repeat(maxMessageWidth - x.messageWidth) + chalk.gray.dim(x.ruleId)
+          ' '.repeat(maxColumnWidth - x.columnWidth) + x.message + '     ' + chalk.gray.dim(x.ruleId),
```

Comment:
The previous formatter adds the rules in the separate line. Changed it to show in the same line as per the screenshots of this library